tests/integration_tests/test_chainermn.py

```diff
@@ -25,6 +25,7 @@
     from typing import Optional  # NOQA
     from typing import Type  # NOQA
 
+    from optuan.integration.chainermn import ChainerMNTrial  # NOQA
```

Comment:
`optuan.integration.chainermn` seems to be a typo for `optuna.integration.chainermn`.

Reply:
Thanks. Fixed: https://github.com/pfnet/optuna/pull/427/commits/734b17eb89ff1b7b43fd0e6a67e069f71b8eea1a